### PR TITLE
Add SAR Buckets and DB

### PR DESCRIPTION
**Pull request by ntsummers1**
### Description

It do what it say

### Related ticket(s)

MDCT-2902

---
### How to test

./dev local
In a separate terminal run `DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin`
Navigate in the browser to localhost:8001
See the tables!
Look in your local uploads folder and see theres a new local-sar-form folder in it!

### Important updates

.env_example was updated. Make sure you update your root .env!

---
### Author checklist

- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [x] I have updated relevant documentation, if necessary

### CODE REVIEW

File: services/database/serverless.yml [hunk 1]
```diff
@@ -127,6 +129,33 @@ resources:
...
+        KeySchema:
+          - AttributeName: state
+            KeyType: HASH
+          - AttributeName: id
+            KeyType: RANGE
+        BillingMode: PAY_PER_REQUEST # Set the capacity to auto-scale
+    SarFormBucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: !Sub ${self:service}-${self:custom.stage}-sar
```
Comment (gmrabian) [inline services/database/serverless.yml:154]: might be safer if this referenced the `sarFormBucket` name from custom. Otherwise that variable isn't used!

**Review by gmrabian — COMMENTED** [review]
looks good overall.

When I started it up locally I saw the db, but didn't see the bucket in my `uploads/local_buckets/` folder. Should I expect that to show up?

File: services/database/serverless.yml (hunk 1 above)
Comment (ntsummers1) [reply services/database/serverless.yml]: Good call on both of these! Great catch. 🙇

**Review by gmrabian — APPROVED** [review]
looks good!

**Review by BearHanded — APPROVED** [review]

**Pull request merged**